local-dev-environment/desktop/radio-service/Cargo.toml

```diff
@@ -8,4 +8,6 @@ edition = "2021"
 [dependencies]
 clap = { version = "4.0", features = ["derive"] }
 tokio = { version = "1.8", features = ["macros", "rt", "rt-multi-thread"] }
```

Comment:
Now that you're moving off of an async runtime, does it make sense to use the std UdpSocket and remove all of tokio?

Reply:
I didn't actually un-async this service in this PR, so the tokio dependencies will stay for now in this one.